src/qibolab/instruments/qblox/q1asm.py

```diff
@@ -0,0 +1,357 @@
+            return col
+
+        max_col: int = 0
+        for line, comment, level in self.lines:
+            if comment:
+                max_col = max(max_col, comment_col(line, level))
+
+        block_str: str = ""
+        if self.name:
+            block_str += (
+                self._indentation_string(self._indentation + Block.GLOBAL_INDENTATION_LEVEL)
+                + "# "
+                + self.name
+                + END_OF_LINE
+            )
+
+        for line, comment, level in self.lines:
+            block_str += self._indentation_string(level + Block.GLOBAL_INDENTATION_LEVEL) + line
+            if comment:
+                block_str += " " * (max_col - comment_col(line, level) + Block.SPACES_BEFORE_COMMENT) + " # " + comment
+            block_str += END_OF_LINE
+
+        return block_str
+
+    def __add__(self, other):
+        if isinstance(other, Block):
+            block = Block()
+            for line, comment, level in self.lines:
+                block.append(line, comment, level)
+            for line, comment, level in other.lines:
+                block.append(line, comment, level)
+        else:
+            raise TypeError(f"Expected Block, got {type(other).__name__}")
+        return block
+
+    def __radd__(self, other):
+        if isinstance(other, Block):
+            block = Block()
+            for line, comment, level in other.lines:
+                block.append(line, comment, level)
+            for line, comment, level in self.lines:
+                block.append(line, comment, level)
+        else:
+            raise TypeError(f"Expected Block, got {type(other).__name__}")
+        return block
+
+    def __iadd__(self, other):
+        if isinstance(other, Block):
+            for line, comment, level in other.lines:
+                self.append(line, comment, level)
+        else:
+            raise TypeError(f"Expected Block, got {type(other).__name__}")
+        return self
+
+
+class Register:
+    """A class to represent a q1asm program register.
+
+    Registers are used as variables in real time FPGA code.
+    Attributes:
+        name (str): a name to identify the register
+    """
+
+    def __init__(self, program: Program, name: str = ""):
+        self._number = program.next_register()
+        self._name = name
+        self._type = type
+
+    def __repr__(self) -> str:
+        return "R" + str(self._number)
+
+    @property
+    def name(self):
+        return self._name
+
+    @name.setter
+    def name(self, value):
+        self._name = value
+
+
+def wait_block(wait_time: int, register: Register, force_multiples_of_4: bool = False):
+    """Generates blocks of code to implement long delays.
+
+    Arguments:
+        wait_time (int): the total time to wait.
+        register (:class:`qibolab.instruments.qblox.qblox_q1asm.Register`): the register used to loop
+        force_multiples_of_4 (bool): a flag that forces the delay to be a multiple of 4(ns)
```

Comment:
@stavros11 I am agree with you. It is a good practice to avoid the numbers in the variable names. I have modified all ocurrencies by force_multiples_of_four